capture_gui/plugins/displayplugin.py

```diff
@@ -0,0 +1,148 @@
+
+TOP_DEFAULT = [0.5350000262260437,
+               0.6169999837875366,
+               0.7020000219345093]
+
+BOTTOM_DEFAULT = [0.052000001072883606,
+                  0.052000001072883606,
+                  0.052000001072883606]
+
+COLORS = {"background": BACKGROUND_DEFAULT,
+          "backgroundTop": TOP_DEFAULT,
+          "backgroundBottom": BOTTOM_DEFAULT}
+
+LABELS = {"background": "Background",
+          "backgroundTop": "Top",
+          "backgroundBottom": "Bottom"}
+# endregion GLOBALS
+
+
+class DisplayPlugin(capture_gui.plugin.Plugin):
+
+    """Plugin to apply viewport visibilities and settings"""
+
+    id = "Display Options"
+    label = "Display Options"
+    section = "config"
+    order = 70
+
+    def __init__(self, parent=None):
+        super(DisplayPlugin, self).__init__(parent=parent)
+
+        self._colors = dict()
+
+        self._layout = QtWidgets.QVBoxLayout()
+        self.setLayout(self._layout)
+
+        self.override = QtWidgets.QCheckBox("Override Display Options")
+
+        self.display_type = QtWidgets.QComboBox()
+        self.display_type.addItems(["Solid", "Gradient"])
+
+        # create color columns
+        self._color_layout = QtWidgets.QHBoxLayout()
+        for label, default in COLORS.items():
+            self.add_color_picker(self._color_layout, label, default)
+
+        # populate layout
+        self._layout.addWidget(self.override)
+        self._layout.addWidget(self.display_type)
+        self._layout.addLayout(self._color_layout)
+
+        self.connections()
+
+        # ensure widgets are in the correct enable state
+        self.toggle_override()
+
+    def connections(self):
+        self.override.toggled.connect(self.toggle_override)
+
+    def add_color_picker(self, layout, label, default):
+        """Create a column with a label and a button to select a color
+        
+        :param layout: the layout to add the color picket too
+        :type layout: QtWidgets.QLayout
+        
+        :param label: the system name for the color type, e.g. : backgroundTop
+        :type label: str
+        
+        :param default: the default color combination to start with
+        :type default: list
+        
+        :return: a color picker instance
+        :rtype: colorpicker.ColorPicker
+        """
+
+        column = QtWidgets.QVBoxLayout()
+        label_widget = QtWidgets.QLabel(LABELS[label])
+
+        color_picker = colorpicker.ColorPicker()
+        color_picker.color = default
+
+        column.addWidget(label_widget)
+        column.addWidget(color_picker)
+
+        column.setAlignment(label_widget, QtCore.Qt.AlignCenter)
+
+        layout.addLayout(column)
+        self._colors[label] = color_picker
+
+        return color_picker
+
+    def toggle_override(self):
+        state = self.override.isChecked()
+        self.display_type.setEnabled(state)
+        for widget in self._colors.values():
+            widget.setEnabled(state)
+
+    def display_gradient(self):
```

Comment:
Let's add a docstring here that explains what the text being equal to "Gradient" means:

```
"""Returns whether the background should be displayed as gradient or solid color.

When True the colors will use the top and bottom color to define the gradient.
Otherwise the background color will be used as solid color.
"""
```

Or what do you think?